README.md

```diff
@@ -40,34 +38,81 @@ Both of these installation methods work if you are using virtualenv, which you s
 
 ### From Pypi
 
-Coming soon
+Coming soon.
 
 
-API Implementation
-------------------
+## API Implementation
 
-Status listing of the HPE OneView REST interfaces that have been implemented in the Python library so far is hosted in the [Wiki section](https://github.com/HewlettPackard/python-hpOneView/wiki/HPE-OneView-API-Version-200-Implementation-Status)
+Status listing of the HPE OneView REST interfaces that have been implemented in the Python library so far is hosted in the [Wiki section](https://github.com/HewlettPackard/python-hpOneView/wiki/HPE-OneView-API-Version-200-Implementation-Status).
```

Comment:
I would change this: "Status listing of the HPE OneView REST interfaces that have been implemented in the Python library so far is hosted in the [Wiki section]" to this: "A status of the HPE OneView REST interfaces that have been implemented in this Python library can be found in the  [Wiki section]"